### PR TITLE
fix #282: single-line comment after case name

### LINKED ISSUE #282: Single-line comments can't appear after case name
Reported by udob on Discord:

> I think I found a problem with line comments in grammars. Checkout this example:
> 
> ```
> Arithmetic {
>   Exp 
>     = AddExp        // line comment OK
> 
>   AddExp
>     = AddExp "+" digit+   -- plus  // line comment after caseName FAILED
>     | AddExp "-" digit+   -- minus
>     | digit+
> }
> ```
> 
> The second line comment produces a syntax error (expected "}" or "\n"). Changing the line comment to a /* ...*/ makes the error go away.

### PR DESCRIPTION
**Pull request by pdubroy**

### CODE REVIEW

File: ohm-js/src/ohm-grammar.ohm [hunk 1]
```diff
@@ -109,7 +109,7 @@ Ohm {
    += comment
 
   comment
-    = "//" (~"\n" any)* "\n"  -- singleLine
+    = "//" (~"\n" any)* &"\n"  -- singleLine
```
Comment (alexwarth) [inline ohm-js/src/ohm-grammar.ohm:112]: Sorry I'm late to the party!

Should we make the look-ahead expression here `&("\n" | end)` to allow single-line comments to happen at the last line of a file that doesn't have a `"\n"` at the end?

Comment (pdubroy) [reply ohm-js/src/ohm-grammar.ohm:112]: Good idea.

Comment (pdubroy) [reply ohm-js/src/ohm-grammar.ohm]: Done.

Comment (alexwarth) [reply ohm-js/src/ohm-grammar.ohm]: Cool, thanks @pdubroy!

**Pull request merged**